tests/test_transforms.py

```diff
@@ -207,23 +220,22 @@ def test_resample(input, input_sample_rate, output_sample_rate, method, axis):
 @pytest.mark.parametrize('input,window_size,hop_size,axis,magnitude_boost', [
     (A, 4, 1, 2, 1e-07),
     pytest.param(A, 4, 1, 2, -1e-07, marks=xfail(raises=ValueError)),
-    pytest.param(A, 4, 1, 2, 0, marks=xfail(raises=ValueError)),
+    pytest.param(A, 4, 1, 2, -1, marks=xfail(raises=ValueError)),
```

Comment:
Why did you change the axis to `-1`?

Reply:
It's not the axis but the `magnitude_boost`. I knew I had forgotten something in the discussion :)

I now allow the `magnitude_boost` to be 0 just in case the user does not want to add any limit to it.

Reply:
OK, makes sense.